pkg/proxy/nftables/helpers_test.go

```diff
@@ -146,11 +146,11 @@ func diffNFTablesChain(nft *knftables.Fake, chain, expected string) string {
 
 // assertNFTablesChainEqual asserts that the indicated chain in nft's table contains
 // exactly the rules in expected (in that order).
-func assertNFTablesChainEqual(t *testing.T, line string, nft *knftables.Fake, chain, expected string) {
-	if diff := diffNFTablesChain(nft, chain, expected); diff != "" {
-		t.Errorf("rules do not match%s:\ndiff:\n%s", line, diff)
-	}
-}
+//func assertNFTablesChainEqual(t *testing.T, line string, nft *knftables.Fake, chain, expected string) {
```

Comment:
should I just remove this function, or keep it (commented) for future use?

Reply:
You can just remove it. I don't currently expect to need it for anything else.